lib/eco/treemanager.py

```diff
@@ -1048,14 +1048,22 @@ def jump_cursor_within_selection(self, key):
             [self.selection_start, self.selection_end])
 
         if key.left:
-            self.cursor = selection_start.copy()
```

Comment:
Maybe it would be good to have a method like `set_to` which does these three lines in:

```python
self.cursor.set_to(selection)
```